### PR TITLE
Create an anonymous func to remove duplicate code.

### PR DESCRIPTION
**Pull request by albertoleal**

### CODE REVIEW

File: android_symbol_uploader.go [hunk 1]
```diff
@@ -142,14 +141,17 @@ func (a *androidSymbolUploader) getBuildVersion(e *env) (int, error) {
...
-	if err != nil {
-		return 0, err
-	}
-	if versionCode == 1 {
-		fmt.Fprintln(e.Out, "Warning: build number is '1'.")
+	var (
+		versionCode int
+		err         error
+	)
+	if a.Manifest != "" {
```
Comment (pavanka) [inline android_symbol_uploader.go]: I would write this part like

if a.Manifest != "" {
  return f(a.getBuildVersionFromManifest)
}
return f(a.getBuildVersionFromAPK())

File: android_symbol_uploader.go [hunk 2]
```diff
@@ -131,8 +131,7 @@ func parseVersionFromBytes(bytes []byte) (int, error) {
 }
 
 func (a *androidSymbolUploader) getBuildVersion(e *env) (int, error) {
-	if a.Manifest != "" {
-		versionCode, err := a.getBuildVersionFromManifest()
+	var f = func(versionCode int, err error) (int, error) {
```
Comment (pavanka) [inline android_symbol_uploader.go]: perhaps name is better like
handleError

File: android_symbol_uploader.go [hunk 3]
```diff
@@ -142,14 +141,11 @@ func (a *androidSymbolUploader) getBuildVersion(e *env) (int, error) {
...
 
-	versionCode, err := a.getBuildVersionFromAPK()
-	if err != nil {
-		return 0, err
-	}
-	if versionCode == 1 {
-		fmt.Fprintln(e.Out, "Warning: build number is '1'.")
+	if a.Manifest != "" {
+		return handleError(a.getBuildVersionFromManifest())
+	} else {
```
Comment (pavanka) [inline android_symbol_uploader.go]: you do not need an else.
if we enter if then we just return

so 
if {
return
}
return

is good. this is idiomatic go :)

Comment (albertoleal) [inline android_symbol_uploader.go]: Great! Just removed that.

**Comment by pavanka** [comment]
lgtm. thanks alberto, I'll merge both later today.

**Pull request closed without merging**